app/serializers/response_serializer.rb

```diff
@@ -15,3 +16,7 @@ class AnswerOptionSetSerializer < ActiveModel::Serializer
 
   has_many :answer_options
 end
+
+class DomainSerializer < ActiveModel::Serializer
```

Comment:
Goed opgemerkt, nu kan hij volgens mij zelfs niet ge-autoload worden als hij wordt aangeroepen vanuit `DomainsController` als `ResponseSerializer` niet eerst is gebruikt.